app/components/FamilyList/FamilyList.tsx

```diff
@@ -1,40 +1,31 @@
-import { FamilySeverityEnum, FamilyStatusEnum } from '@camp/data-layer';
-import { DashboardCard } from '@camp/design';
-import { messages } from '@camp/messages';
+import { useFamilyListQuery } from '@camp/data-layer';
+import { DashboardCard, FullPageLoader, showNotification } from '@camp/design';
+import { errorMessages, messages } from '@camp/messages';
+import { createTestAttr } from '@camp/test';
+import { isEmpty, isNull } from '@fullstacksjs/toolbox';
 import { Title } from '@mantine/core';
 
 import { CreateFamilyButton } from '../CreateFamily';
+import { FamilyEmptyState } from '../FamilyEmptyState';
+import * as ids from './FamilyList.ids';
 import { FamilyTable } from './FamilyTable';
-import type { ShortFamiliesInfo } from './FamilyTable/toShortFamilyInfoTableRows';
-
-/* FIXME
-  this type should be removed in
-  backend integration ticket of FamilyList
-*/
-export const shortFamiliesInfo: ShortFamiliesInfo = [
-  {
-    id: '(F00001)',
-    name: 'فول استک زاده',
-    severity: FamilySeverityEnum.Critical,
-    status: FamilyStatusEnum.Completed,
-  },
-  {
-    id: '(F00002)',
-    name: 'فول استک زاده',
-    severity: FamilySeverityEnum.Critical,
-    status: FamilyStatusEnum.Draft,
-  },
-  {
-    id: '(F00003)',
-    name: 'فول استک زاده',
-    severity: FamilySeverityEnum.Normal,
-    status: FamilyStatusEnum.Completed,
-  },
-];
 
 export const FamilyList = () => {
   const t = messages.families.list;
+  const { data, loading, error } = useFamilyListQuery();
+  const families = data?.family;
+
+  if (error)
+    showNotification({
+      type: 'failure',
+      title: t.title,
+      message: errorMessages.UNKNOWN_ERROR,
+      ...createTestAttr(ids.familyListFailureNotification),
+    });
```

Comment:
What to render?

Reply:
we show the notification if there was an error
and since there is an error the families must be null so it's handled after that

Reply:
What if we return a JSX here?